qemu/tests/cfg/s390x_cpu_model.cfg

```diff
@@ -0,0 +1,41 @@
+- s390x_cpu_model:
+    auto_cpu_model = yes
+    start_vm = yes
+    kill_vm_on_error = yes
+    take_regular_screendumps = no
+    virt_test_type = qemu
+    only s390x
+    variants:
+        - expansion:
+            type = s390x_cpu_model_expansion
+            variants:
+               - model_type_z196_z196_2_z114:
+                    cpu_models = "z196 z196.2 z114"
+                    'name1=value1 name2=value2 ...'
+                    props = 'aefsi=True msa4=True msa3=True msa2=True msa1=True sthyi=True edat=True ipter=True bpb=True ppa15=True cmm=True'
+               - model_type_zEC12_zEC12_2:
+                    cpu_models = "zEC12 zEC12.2 zBC12"
+                    props = 'aen=True aefsi=True msa4=True msa3=True msa2=True msa1=True sthyi=True edat=True ri=True edat2=True ipter=True esop=True cte=True bpb=True ppa15=True zpci=True sea_esop2=True te=True cmm=True'
+               - model_type_z13_z13_2_z13s:
+                    cpu_models = "z13 z13.2 z13s"
+                    props = 'aen=True aefsi=True msa5=True msa4=True msa3=True msa2=True msa1=True sthyi=True edat=True ri=True edat2=True vx=True ipter=True esop=True cte=True bpb=True ppa15=True zpci=True sea_esop2=True te=True cmm=True'
+               - model_type_z14_z14_2_z14ZR1:
+                    cpu_models = "z14 z14.2 z14ZR1"
+                    props = 'aen=True aefsi=True mepoch=True msa8=True msa7=True msa6=True msa5=True msa4=True msa3=True msa2=True msa1=True sthyi=True edat=True ri=True edat2=True vx=True ipter=True mepochptff=True vxeh=True vxpd=True esop=True iep=True cte=True bpb=True gs=True ppa15=True zpci=True sea_esop2=True te=True cmm=True'
+               - model_type_gen15a_gen15b:
+                    cpu_models = "gen15a gen15b"
+                    props = 'aen=True vxpdeh=True aefsi=True mepoch=True msa9=True msa8=True msa7=True msa6=True msa5=True msa4=True msa3=True msa2=True msa1=True sthyi=True edat=True ri=True deflate=True edat2=True etoken=True vx=True ipter=True mepochptff=True vxeh=True vxpd=True esop=True msa9_pckmo=True vxeh2=True iep=True cte=True gs=True ppa15=True zpci=True sea_esop2=True te=True cmm=True'
+               - model_type_gen16a_gen16b:
+                    cpu_models = "gen16a gen16b"
+                    props = 'nnpa=True aen=True vxpdeh=True aefsi=True mepoch=True msa9=True msa8=True msa7=True msa6=True msa5=True msa4=True msa3=True msa2=True msa1=True sthyi=True edat=True ri=True deflate=True edat2=True etoken=True vx=True ipter=True pai=True mepochptff=True vxeh=True vxpd=True esop=True msa9_pckmo=True vxeh2=True iep=True cte=True gs=True ppa15=True zpci=True rdp=True sea_esop2=True beareh=True te=True cmm=True vxpdeh2=True'
+        - boot_cpu_models:
+            type = s390x_cpu_model_boot
+            auto_cpu_model = yes
+            cpu_model_check_cmd = "query-cpu-model-expansion"
+            cpu_model_check_args = '{"type": "static", "model": {"name": "host"}}'
+            Host_RHEL.m7:
```

Comment:
@fbq815 I believe a 7.9 guest should also start when given a z15 cpu model. Same for a RHEL 8 host, it should start on gen16.

Reply:
Added